### ISSUE TITLE
Terraform AWS default tags detected as drift

### ISSUE THREAD

**Issue by mwarkentin** [cloudskiff/driftctl#530]
**Description**
We recently started using the new [Terraform AWS Default Tags feature](https://www.hashicorp.com/blog/default-tags-in-the-terraform-aws-provider) which allows you to define default tags in the provider which will be automatically added to all supported resources.

driftctl appears to be detecting tags added this way as drift:

```
Found changed resources:
  - devjs-account (aws_dynamodb_table):
    + Tags.Application: <nil> => "foo"
    + Tags.Env: <nil> => "devjs"
```

**Environment**
 - OS: MacOS
 - driftctl version: 0.8

**How to reproduce**

```
provider "aws" {
  version = "~> 3.0"
  region  = "us-east-1"

  assume_role {
    role_arn     = "arn:aws:iam::***********:role/**********"
  }

  default_tags {
    tags = {
      Application = "foo"
      Env         = local.environment
    }
  }
}

module "ddb_foo" {
  source  = "cloudposse/dynamodb/aws"
  version = "0.26.0"

  stage     = local.environment
  name      = "foo"
  hash_key  = "fooId"

  billing_mode = "PAY_PER_REQUEST"
}
```

**Possible Solution**

**Additional context**

**Comment by sjourdan** [comment]
@mwarkentin - this one will be split in two: 

- https://github.com/cloudskiff/driftctl/issues/679 to first ignore those tags as they create too much noise (released in 0.10)
- https://github.com/cloudskiff/driftctl/issues/635 to process them correctly (probably in 0.11)

**Comment by sjourdan** [comment]
fixed by 0.10.0 release! https://github.com/cloudskiff/driftctl/releases/tag/v0.10.0

subscribe to https://github.com/cloudskiff/driftctl/issues/635 for a future better processing
